test/index.ts

```diff
@@ -57,7 +75,7 @@ describe('Queries Test', () => {
 });
 
 describe('Mutation Test', () => {
-  it('Create Mutation', async () => {
+  it('Correct Create Mutation', async () => {
```

Comment:
```suggestion
  it('should create user successfully', async () => {
```

O enunciado nesse formato (começando com should) é uma convenção da comunidade pra escrita de testes